docfx_project/api/rest/index.md

```diff
@@ -1,28 +1,34 @@
 
 # Ziti controlller REST API
 
-The API version and Swagger definition for a particular controller may be fetched and there are relatively good documents
+The API version and Swagger (OpenAPI 2.0) definition for a particular controller may be fetched and there are relatively good documents
 available as well, also served directly from the controller. There are two main endpoints 
 
-## Client API
+## Edge-Client API
 
-The client api services are used by endpoints to interact with the [Ziti Network](~/ziti/overview.md#overview-of-a-ziti-network).
-These services are necessary to authenticate, enroll, return services and fetch other information related to the 
+The edge-client API services are used by endpoints to interact with the [Ziti Network](~/ziti/overview.md#overview-of-a-ziti-network).
+These services are necessary to authenticate, enroll, return services and fetch other information related to the
 endpoint's participating in the network.
 
-### Download the Client API Spec
+### Download the Edge-Client API Spec
 
-The client api spec can be downloaded from your controller at the following url (replace "${CONTROLLER_URL}" accordingly)
-[https://${CONTROLLER_URL}/edge/client/v1/docs](https://${CONTROLLER_URL}/edge/client/v1/docs).
+The edge-client API spec for your running controller is published as part of the API itself and may be fetched from [https://${CONTROLLER_ADDRESS}/edge/client/v1/docs](https://${CONTROLLER_ADDRESS}/edge/client/v1/docs). The latest edge-client API spec for is also available from [the Git repo](https://github.com/openziti/edge/blob/main/specs/client.yml).
 
 ## Management API
 
-The management api is used to configure, manage, and maintain the [Ziti Network](~/ziti/overview.md#overview-of-a-ziti-network).
+The management API is used to configure, manage, and maintain the [Ziti Network](~/ziti/overview.md#overview-of-a-ziti-network).
 
 ### Download the Management API Spec
 
-The client api spec can be downloaded from your controller at the following url (replace "${CONTROLLER_URL}" accordingly)
-[https://${CONTROLLER_URL}/edge/management/v1/docs](https://${CONTROLLER_URL}/edge/management/v1/docs).
+The management API spec too may be downloaded for your running controller at [https://${CONTROLLER_ADDRESS}/edge/management/v1/docs](https://${CONTROLLER_ADDRESS}/edge/management/v1/docs). The latest management API spec is also available from [the Git repo](https://github.com/openziti/edge/blob/main/specs/management.yml).
+
+## The Fabric Management API
+
+The fabric API is a lesser concern for most users, implementers, and integrators because it is itself implemented by the Ziti Edge and so most fabric-level operations are performed on your behalf in the course of using Ziti.
+
+### Download the Fabric Management API Spec
+
+The fabric API spec may be fetched from [the Git repo](https://github.com/openziti/fabric/blob/main/specs/swagger.yml).
```

Comment:
i really think it's just too early to put the fabric stuff in this doc for the same reason i don't think it's a good idea to point to the spec from the repo. it can be old/out of date. i suppose it's fine to mention there's a fabric api - but even that feels just premature to me.  i'd rather put in an issue to document all this better from a more wholistic, comprehensive point of view.

Reply:
For the reasons mentioned in the disclaimer in this diff I care least about Fabric API spec of the three specs and would not shed a tear if we only advertised the latest released spec from the `main` branch for Edge Client and Management API's respective repos.

Reply:
I've commented the link to the published Fabric API spec because it is the least important of the three for the purposes of this document.